framework/src/modules/chain/components/storage/entities/account.js

```diff
@@ -111,7 +111,7 @@ class ChainAccount extends AccountEntity {
 		return accounts;
 	}
 
-	async update(filters, data, _options, tx) {
+	update(filters, data, _options, tx) {
```

Comment:
I think all the functions which return Promise are better to be `async` since it's more clear that it returns `Promise`

Reply:
I get the point but there's some overhead associated with using async vs regular promises so though I get the point but maybe we better keep it when really necessary?

Reply:
Although it seems the overhead is about 1% so maybe readability is better in this case...

Reply:
I think that doesn't apply to something that is already returning promise? Or does it apply all the time?

Reply:
All the time as async wraps promises. But 1% is small enough to be disregarded.

Reply:
Mmm then add long as it's consistent I'm ok for now